Dropdown/Dropdown.js

```diff
@@ -191,6 +191,9 @@ const DropdownBase = kind({
 		onOpen: handle(
 			forProp('open', false),
 			forward('onOpen')
+		),
+		onScrollerClick: handle(
```

Comment:
Just note: actually we don't need `handle` for this as we don't use any props and no chaining functions here.  But it looks not bad to me.